plugins/chat/spec/integration/outgoing_web_hooks_spec.rb

```diff
@@ -102,20 +102,12 @@ def expect_web_hook_payload_channel_to_match_direct_message(channel:, direct_mes
     context "for a category channel" do
       fab!(:category)
       fab!(:chat_channel) { Fabricate(:category_channel, chatable: category) }
-      fab!(:chat_message) { Fabricate(:chat_message, chat_channel: chat_channel, user: user1) }
-
-      before do
-        [user1, user2].each do |user|
-          Chat::UserChatChannelMembership.create(
```

Comment:
Removed this as it's causing a duplicate key violation, I think when we use the service this doesn't need to be done manually

Reply:
yes using Chat::CreateMessage will add you to channel if you are not